.circleci/config.yml

```diff
@@ -80,7 +80,7 @@ jobs:
           file: coverage.txt
   check-lint:
     executor: golang
-    resource_class: large
+    resource_class: xlarge
```

Comment:
The linter is failing because of OOM

Reply:
If we run into issues in the future maybe we can look into splitting up the circle jobs a bit so that they don't all run at the same time. FTR I don't know if that would help but it might be worth looking into.